dist/description/description-en.txt

```diff
@@ -15,26 +15,26 @@ Out of the box, these lists of filters are loaded and enforced:
 - EasyPrivacy
 - Malware domains
 
-More lists are available for you to select if you wish:
+More lists are available for you to select, should you wish to:
 
 - Fanboy’s Enhanced Tracking List
-- Dan Pollock’s hosts file
-- hpHosts’s Ad and tracking servers
+- Dan Pollock's hosts file
+- hpHosts's Ad and tracking servers
 - MVPS HOSTS
 - Spam404
 - And many others
 
-Of course, the more filters enabled, the higher the memory footprint. Yet, even after adding Fanboy's two extra lists, hpHosts’s Ad and tracking servers, uBlock Origin still has a lower memory footprint than other very popular blockers out there.
+Of course, the more filters are enabled, the higher the memory footprint. Yet, even after adding Fanboy's two extra lists, hpHosts’s Ad and tracking servers, uBlock Origin still has a lower memory footprint than other popular blockers out there.
 
-Also, be aware that selecting some of these extra lists may lead to higher likelihood of web site breakage -- especially those lists which are normally used as hosts file.
+Also, be aware that selecting some of these extra lists may lead to higher likelihood of website breakage—especially those lists which are normally used as the hosts file.
 
 ***
 
-Free.
-Open source with public license (GPLv3)
+Gratis.
```

Comment:
Really?

Reply:
Free is ambiguous, as it could be either _gratis_ or _free_ (as in free software).
http://www.dictionary.com/browse/gratis

Reply:
... but it's both, so why remove the licence mention?

Reply:
@CrisBRM I didn't, it is specified further down. :)
> +Copyleft libre software license (GPLv3)